ansible-collection-microsoft-sql/ansible-collection-microsoft-sql.spec

```diff
@@ -227,27 +105,14 @@ BuildRequires: python3dist(ruamel.yaml)
 Requires: python3-jmespath
 Requires: python3-netaddr
```

Comment:
Is `python3-netaddr` needed?

Reply:
Hi Noriko, thank you for the review.

I didn't know that, I will remove these requirements now. The mssql role does not have any requirements.